package.json

```diff
@@ -178,7 +178,8 @@
 			"id": "ruby",
 			"aliases": ["Ruby", "ruby"],
 			"extensions": [".rb", ".rbx", ".rjs", ".Rakefile", ".rake", ".cgi", ".fcgi", ".gemspec", ".irbrc", ".capfile",
-				".ru", ".prawn", ".Gemfile", ".Guardfile", ".Vagrantfile"]
+				".ru", ".prawn", ".Gemfile", ".Guardfile", ".Vagrantfile"],
+			"configuration": "./language-configuration.json"
```

Comment:
This isn't required because we load the language configuration in the code. This would also require maintaining the same code in multiple locations. It's loaded in the code because we use RegExp for some of it and that's more than a pain to do this way.